files/日本語 Vocab - Grammar.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
This is a dumb sample that I built for testing, the real load file will take a lot more work!